src/remotestorage.js

```diff
@@ -763,8 +763,12 @@
     },
 
     localStorageAvailable: function () {
```

Comment:
Can't we reuse `RemoteStorage.LocalStorage._rs_supported` or even not have to check that feature since it was initialized already?